tests/benchmark_utf8.cpp

```diff
@@ -57,8 +57,10 @@ int main() {
   double t = 0;
   double t2 = emscripten_get_now();
   for(int i = 0; i < 100000; ++i) {
-    // FF Nightly: Already on small strings of 64 bytes in length, TextDecoder trumps in performance.
-    char *str = randomString(8);
+    // Create strings of lengths 1-32, because the internals of text decoding
+    // have a cutoff of 16 for when to use TextDecoder, and we wish to test both
+    // (see UTF8ArrayToString).
+    char *str = randomString((rand() % 32) + 1);
```

Comment:
Can we avoid using rand() in tests?   How about `i % 32` instead?

Reply:
There is already another `rand()` usage in this file, on line 40. There is an `srand` so it should be deterministic. Is that ok with you? If not I can look into a larger refactor here for both of them (`i` is not accessible in the other place, so it's not trivial).

Reply:
Yeah I did notice the other one... I don't think we need to fix that now, but can we just avoid the introduction of another one.. since its easy to use i here?